how-to/databases/install-postgresql.md

```diff
@@ -30,7 +30,7 @@ listen_addresses = '*'
 ```
 
 ```{note}
-'*' will allow all available IP interfaces (IPv4 and IPv6), to only listen for IPv4 set `0.0.0.0` while '`::`' allows listening for all IPv6 addresses.
+'`*`' will allow connections from all available IP interfaces (IPv4 and IPv6). To only listen for IPv4 set `listen_addresses` to '`0.0.0.0`', while '`::`' allows listening for all IPv6 addresses.
```

Comment:
This is still confusing (as was the original text). Let's unpack it.

First, `listen_addresses`  is not about "allowing" connections "from" all available IP interfaces. This is about where the postgresql daemon will listen on.

So I would say that `*` will have postgresql listening on all available network interfaces on that system (IPv4 or IPv6, doesn't matter). The distinction about IPv4 and IPv6 sounds correct, and your text is better.